### ISSUE TITLE
Consider inlining `carbon-icons-svelte` dependency

### ISSUE THREAD

**Issue by metonym** [carbon-design-system/carbon-components-svelte#884]
Currently, this library has `carbon-icons-svelte` as a direct dependency.

There are over 29,000 files in `carbon-icons-svelte`; only a few dozen of those are actually used by the components.

As a consumer, I don't want to download extra files if I am only interested in components. It can always be installed separately.

By "inlining," only the icons used by this library should be shipped in the NPM package to keep it as lightweight as possible.

**Comment by metonym** [comment]
Released in [v0.48.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.48.0)

**Comment by metonym** [comment]
Download speeds before/after removing `carbon-icons-svelte`.

This is using Yarn and an internet connection with a ~110 Mbps download speed. The results are non-scientific.

**Before**

```sh
yarn add -D carbon-components-svelte@0.47.6

✨  Done in 5.95s.
```

**After**

```sh
yarn add -D carbon-components-svelte@0.48.0

✨  Done in 1.93s.
```
